git-changebar/src/gcb-plugin.c

```diff
@@ -878,6 +878,10 @@ get_widget_for_buf_range (GeanyDocument *doc,
                                MIN (width + 2, alloc.width),
                                MIN (height + 1, alloc.height));
   
+  /* Size request seems to scroll Scintilla view so we have to re-set visible lines again */
+  scintilla_send_message (sci, SCI_SETENDATLASTLINE, 0, 0);
```

Comment:
any reason to re-set `ENDATLASTLINE`?  I don't see it being needed on my end, did you see an issue without it?

Reply:
No, it doesn't seem to be necessary, I just copied the two lines above. Repushed without this line.